### ISSUE TITLE
[SideNavigationItem][A11y]: Duplicate text announced by screen readers for some icons in SideNavigationItem

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11304]
### Bug Description

When using the `SideNavigationItem` component with certain icons, such as `action-settings`, screen readers (JAWS and NVDA) announce the element's text twice. For instance, the element is read as "settings settings". This issue occurs because some icons include an aria-label attribute, which causes redundant text to be read by screen readers.

<SideNavigationItem
  icon="action-settings"
  text="Settings"
  onClick={() => {}}
  selected={}
  onSelect={() => {}}
/>

### Affected Component

SideNavigationItem 

### Expected Behaviour

Screen readers should announce the element's text only once, e.g., "Settings", without duplicating the name from the icon.

### Steps to Reproduce

1. Use the `SideNavigationItem` with an icon "action-settings".
2. Open the application in a web browser.
3. Launch a screen reader (e.g., JAWS or NVDA).
4. Navigate to the `SideNavigationItem` in the UI by using navigation keys.
5. Observe how the screen reader announces the element:
    Expected: "Settings" (once).
    Actual: "Settings settings" (text is duplicated).
6. Test with other icons in the `SideNavigationItem` to see if the issue persists.

### Priority

None

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

Could you please look into that ACC issue ?

Regards,
Plamen Ivanov

**Comment by LidiyaGeorgieva** [comment]
Hello @AlekseiSch ,

I could not reproduce the issue in isolated example - [link](https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cblxuICAgIGh0bWwge1xuICAgICAgZm9yY2VkLWNvbG9yLWFkanVzdDogbm9uZTtcbiAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1zaWRlLW5hdmlnYXRpb24gPiAgXG5cdFx0PHVpNS1zaWRlLW5hdmlnYXRpb24taXRlbSB0ZXh0PVwiSG9tZVwiIGljb249XCJhY3Rpb24tc2V0dGluZ3NcIj48L3VpNS1zaWRlLW5hdmlnYXRpb24taXRlbT5cblx0XHQ8dWk1LXNpZGUtbmF2aWdhdGlvbi1pdGVtIHNsb3Q9XCJmaXhlZEl0ZW1zXCIgdGV4dD1cIkhpc3RvcnlcIiBcbiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICBpY29uPVwiaGlzdG9yeVwiIHRvb2x0aXA9XCJ0b29sdGlwIDQ1NlwiPjwvdWk1LXNpZGUtbmF2aWdhdGlvbi1pdGVtPlxuICAgIDwvdWk1LXNpZGUtbmF2aWdhdGlvbj5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZCAtLT5cbiAgICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCJtYWluLmpzXCI-PC9zY3JpcHQ-XG48L2JvZHk-XG5cbjwvaHRtbD5cbjwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cbiAgIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb24uanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1NpZGVOYXZpZ2F0aW9uSXRlbS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb25Hcm91cC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb25TdWJJdGVtLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9TaWRlTmF2aWdhdGlvbkdyb3VwLmpzXCI7XG5cbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2FjdGlvbi1zZXR0aW5ncy5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvY2hhaW4tbGluay5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvZ3JvdXAuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2xvY2F0ZS1tZS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvY2FsZW5kYXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2hpc3RvcnkuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2N1c3RvbWVyLmpzXCI7In0sIm1haW4uY3NzIjp7Im5hbWUiOiJtYWluLmNzcyIsImNvbnRlbnQiOiJ1aTUtc2lkZS1uYXZpZ2F0aW9uIHtcbiAgICBoZWlnaHQ6IDYwMHB4O1xufSJ9fQ).
Jaws output:
"Navigation List Tree
Home
To move through or expand items use the Arrow keys"

Best Regards,
Lidiya

**Comment by AlekseiSch** [comment]
It seems that this bug was fixed in the latest version. Thanks, I'll close it
